modules/cram/templates/spectre_call.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -euo pipefail
+
+mosdepth () {
+    local args=()
```

Comment:
add note `# Spectre recommends running Mosdepth with a bin size of 1kb and a mapping quality of at least 20` so we won't forget in case we replace `mosdepth` and `samtools coverage` code paths in the future.